src/behavioural-events/tracked-events.ts

```diff
@@ -56,6 +58,16 @@ export interface CheckoutSessionStartEvent extends IEvent {
   };
 }
 
+export interface CheckoutSessionEndEvent extends IEvent {
+  eventName: TrackedEventName.CheckoutSessionEnd;
+  properties: {
+    outcome: "finished" | "closed" | "errored";
```

Comment:
Hmm I'm wondering if this should be 3 separate events, instead of a property with 3 values... Since we have properties that are only needed for specific results. You might have talked about this already, in which case feel free to ignore.

Reply:
Good call @tonidero .

I took a bit of a middle-ground solution here and did not go all the way through. I think it is clearer from a user analytics perspective to have a `CheckoutSessionStart` and `CheckoutSessionEnd` with its outcome. But I have  tighten the types so that they are clearer and narrower on the SDK side.

Let me know if you had concerns on using the same type with different outcomes for the tracked event sent to the backend or this looks good to you otherwise.

Thanks!